assets/js/page-enhancements.js

```diff
@@ -80,33 +80,8 @@
       })
   })
 })($, document)
-;(autoDownloadCoversheetPdf = ($, document) => {
```

Comment:
Do we not want to auto download anymore?

Reply:
Whoops!

Reply:
👨🏻‍🦯